public/index.html

```diff
@@ -47,6 +47,70 @@ <h3>4. Get it fixed</h3>
       <p>Celebrate as maintainters and experts come to your aid with solutions and support.</p>
     </section>
   </div>
+  <div class="faq-container">
+    <section class="faq-item">
 
+      <h3 class="accordion">
```

Comment:
A few notes here:
- The `h3` tag should include only the text itself. The `svg` should be outside the `h3`. You can wrap both in a `header` or `div` tag.
- The class name `accordion` is misleading because it's only the accordion header. The `faq-item` is an accordion item and `faq-container` is the actual accordion. Naming is very important to code readability.

Reply:
move svgs out of heading elements. adjusted class names as suggested